
CaptureRobotPixelsActionGoal action_goal
CaptureRobotPixelsActionResult action_result
CaptureRobotPixelsActionFeedback action_feedback
